src/plugins/utilities/typography/styles.js

```diff
@@ -38,53 +38,61 @@ module.exports = function () {
       },
       '.intro-text': {
         ...theme('decanter.base.fontSmoothing'),
-        fontSize: '1.56em',
-        fontWeight: theme('fontWeight.regular'),
-        lineHeight: '1.5',
-        maxWidth: '85rem',
-        letterSpacing: '-0.012em',
+        ...theme('decanter.typography.type1'),
+        lineHeight: theme('lineHeight.DEFAULT'),
+        maxWidth: theme('maxWidth.prose'),
+        '@screen md': {
+          ...theme('decanter.typography.type2'),
+        },
       },
       '.font-splash': {
         ...headings,
-        marginBottom: '0',
-        fontSize: '2.59em',
-        letterSpacing: '-0.016em',
-        '@screen md': {
-          fontSize: '3.05em',
-        },
+        ...theme('decanter.typography.type6'),
       },
       '.caption': {
         ...theme('decanter.base.fontSmoothing'),
         fontSize: '0.9em',
-        lineHeight: '1.3',
+        lineHeight: theme('lineHeight.snug'),
         color: theme('colors.cool-grey'),
       },
       '.credits': {
         ...theme('decanter.base.fontSmoothing'),
         fontSize: '0.9rem',
-        lineHeight: '1.3',
+        lineHeight: theme('lineHeight.snug'),
         color: theme('colors.cool-grey'),
         fontStyle: 'italic',
       },
       '.subheading': {
-        fontSize: '1.25em',
+        ...theme('decanter.typography.type1'),
       },
       '.quote-text': {
         fontSize: '1.25em',
         fontStyle: 'italic',
       },
       '.big-paragraph': {
+        fontSize: '1.8rem',
+        lineHeight: theme('lineHeight.cozy'),
         '@screen md': {
-          fontSize: '2.1rem',
-          lineHeight: '1.7',
+          fontSize: '2.3rem',
+          lineHeight: theme('lineHeight.DEFAULT'),
         },
       },
-      '.small-paragraph': {
+      '.legacy-body': {
```

Comment:
If we want to keep this size I'd suggest naming it something other than legacy so we are not stuck with deprecating it if it sticks around. Please choose a different name.

Reply:
I think I got rid of this one already. Added the .su-body-basefont-19, 20, 22, 23 as you suggested. Kerri wants 21 as the default font size.